packages/components/Atoms/Grid/__tests__/col.test.js

```diff
@@ -0,0 +1,51 @@
+import { createLocalVue, mount } from '@vue/test-utils'
+import RplCol from './../Col.vue'
+import RplGlobal from '@dpc-sdp/ripple-global'
+
+const localVue = createLocalVue()
+
+localVue.use(RplGlobal, { isDev: true })
+
+const colNotCatchError = mount(RplCol)
+
+const colCatchErrorDev = mount(RplCol, {
+  propsData: {
+    catchChildError: true
+  },
+  localVue
+})
+
+const colCatchErrorNotDev = mount(RplCol, {
+  propsData: {
+    catchChildError: true
+  }
+})
+
+// Give an error for testing.
+// Ideally will be mount an error child component but Vue test utils didn't support mount in slot.
```

Comment:
https://vue-test-utils.vuejs.org/api/options.html#slots

Reply:
`slots` is ok but it won't mount the child. Here is the issue comment from the vue test utils author:
> There isn't currently a way to pass mounted components to slots...
> We aren't going to implement this at the moment. You can pass a component, but not an instance.
https://github.com/vuejs/vue-test-utils/issues/41#issuecomment-327141769 
I did some try from others solution, but I couldn't get error thrown from child in tests.